core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -293,13 +293,12 @@ public T enableFullStreamDecompression() {
    * <p>This method is advisory, and implementations may decide to not enforce this.  Currently,
    * the only known transport to not enforce this is {@code InProcessTransport}.
    *
-   * @param max the maximum number of bytes a single message can be.
-   * @throws IllegalArgumentException if max is negative.
+   * @param bytes the maximum number of bytes a single message can be.
    * @return this
+   * @throws IllegalArgumentException if max is negative.
    * @since 1.1.0
    */
-  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2307")
-  public T maxInboundMessageSize(int max) {
+  public T maxInboundMessageSize(int bytes) {
     // intentional nop
```

Comment:
Probably should add in a checkArgument for bytes >= 0.

Reply:
Done